packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -354,8 +358,11 @@ export class VaModal {
     // Prevents scrolling outside modal
     disableBodyScroll(this.el);
 
+    // Get the docs-root element (from Storybook) if present, so that it can be excluded
+    const docsRoot = document.querySelector('#docs-root');
```

Comment:
When I remove this `docsRoot` (including from the `hideExceptions` assignment), the analytics tests pass so my guess is that it's causing a null error within the testing environment and stopping the event code block below from firing.

But maybe a better question is to ask if we should be putting code into components that are specifically for Storybook? This seems like it might be an anti-pattern.

Reply:
I couldn't find a way to mitigate this issue in the storybook story itself, but did find a way to get this same effect without breaking the automated tests.

Reply:
@Andrew565 I trust what you're saying I'm just still a little bit hung up that we have to add Storybook specific logic to the component. The prop `ariaHiddenNodeExceptions` was supposed to allow for managing conditions like this.

It should be used with the React binding version of the web component (`VaModal`) though. Maybe that's why you said you were just getting a string and not the node?

So in `va-crisis-line-modal` it would be:

```js

const docsRoot = document.querySelector('#docs-root');

  <VaModal
    ariaHiddenNodeExceptions={[docsRoot]}
    modalTitle="We’re here anytime, day or night – 24/7"
    onPrimaryButtonClick={() => this.setNotVisible()}
    onCloseEvent={() => this.setNotVisible()}
    visible={this.isOpen}
    large={true}
    uswds={true}
  >
```

I realize this kind of just moves Storybook specific stuff into another component but at least we aren't changing logic in the modal component itself and the prop is being used as it's intended.